packages/worker/src/api/controllers/global/configs.ts

```diff
@@ -126,10 +127,10 @@ export async function save(ctx: UserCtx<Config>) {
   const existingConfig = await configs.getConfig(type)
   let eventFns = await getEventFns(ctx.request.body, existingConfig)
 
-  // Config does not exist yet
-  if (!existingConfig) {
-    body._id = configs.generateConfigID(type)
+  if (existingConfig) {
+    body._rev = existingConfig._rev
```

Comment:
What is the use case of this? If the `_rev` value is used to avoid unexpected overrides while multiple processes/users working in parallel, will this not invalidate this safety check?